### DISCUSSION TITLE
This condition is always false.

### DISCUSSION THREAD

**Discussion by llchry** [minio/minio#17464] [Q&A]
https://github.com/minio/minio/blob/1443b5927a83868f2888eb7b26abe70e6f773a94/internal/config/storageclass/storage-class.go#LL170C2-L173C3

https://github.com/minio/minio/blob/1443b5927a83868f2888eb7b26abe70e6f773a94/internal/config/storageclass/storage-class.go#LL186C2-L189C3

https://github.com/minio/minio/blob/1443b5927a83868f2888eb7b26abe70e6f773a94/internal/config/storageclass/storage-class.go#LL193C2-L195C3

This condition is always false. Can this line of code be deleted?

**Comment by klauspost** [comment]
Yes. This is from before we allowed objects to have no parity.

**Reply by llchry** [reply]
So who's going to submit mr to delete these codes?
